components/Mermaid/index.tsx

```diff
@@ -10,35 +10,34 @@ let mermaid: MermaidType;
 
 const Mermaid = ({ children }: React.PropsWithChildren) => {
   const theme = useContext(ThemeContext);
-  const containerRef = useRef<HTMLDivElement>(null);
+  const [content, setContent] = useState('');
```

Comment:
Maybe just set the children as the default here?
```suggestion
  const [content, setContent] = useState(props.children);
```

Reply:
Oh sorry, just `useState(children)`